appeals/api/src/server/repositories/service-user.repository.js

```diff
@@ -0,0 +1,21 @@
+import { databaseConnector } from '#utils/database-connector.js';
+
+/** @typedef {import('@pins/appeals.api').Schema.ServiceUser} ServiceUser */
+/**
+ * @typedef {import('#db-client').Prisma.PrismaPromise<T>} PrismaPromise
+ * @template T
```

Comment:
No harm to leave this, but PrismaPromise and Promise are compatible, so no need to import this every time you can just do `Promise<Type>` and it should still be happy.